### DISCUSSION TITLE
Not available on M1 Mac

### DISCUSSION THREAD

**Discussion by jkfor6** [thygate/stable-diffusion-webui-depthmap-script#178] [Q&A] [answered]
M1 Macbook 
Even simple depth output can't be generated.

> DepthMap v0.3.11
> device: cpu
> Loading model weights from  ./models/leres/res101.pth
> Attempting to deserialize object on a CUDA device but torch.cuda.is_available() is False. If you are running on a CPU-only machine, please use torch.load with map_location=torch.device('cpu') to map your storages to the CPU.
> All done.

**Comment by thygate** [comment, marked as answer]
Select CPU to compute on non cuda devices for now. Nowhere it is advertised that gpu is supported by this extension on mac silicon.

**Reply by jkfor6** [reply]
> Select CPU to compute on non cuda devices for now. Nowhere it is advertised that gpu is supported by this extension on mac silicon.

Thanks, I'll try it

**Reply by thygate** [reply]
It could be pretty straightforward to support MPS, I just have no way of testing it.
